parsers/CA_AB.py

```diff
@@ -81,7 +81,9 @@ def fetch_price(
         if row[1] != "-":
             prices.append(
                 zoneKey=zone_key,
-                datetime=arrow.get(row[0], "MM/DD/YYYY HH", tzinfo=TIMEZONE).datetime,
+                datetime=datetime.strptime(row[0], "%m/%d/%Y %H").replace(
+                    tzinfo=TIMEZONE
+                ),
```

Comment:
This fails with:
```python
    raise ValueError("unconverted data remains: %s" %
ValueError: unconverted data remains: 4
```
It will needs to be addressed before we can merge it.

Reply:
Fixed ValueError by converting %H to 00 index.